modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/http/ESHttpClient.java

```diff
@@ -106,9 +217,14 @@ public String getClusterName() {
 		String clusterName;
 		try {
 			JestResult result = client.execute(nodesinfo);
-			clusterName = result.getValue("cluster_name").toString();
+			if (result == null ) {
```

Comment:
No: this is when we cannot read cluster name from REST (e.g. 403). We then use user provided one